packages/Input/Input.jsx

```diff
@@ -23,7 +23,12 @@ const Input = props => (
       <div className={positionStyles.relative}>
         <input
           {...inputProps}
-          className={joinClassNames(className, formStyles.height, styles.hideNumberSpinner)}
+          className={joinClassNames(
+            className,
+            formStyles.height,
```

Comment:
This can be removed since it sets `height: 3.25rem` that you intend to set to `height: auto` using `styles.input`